cnect-web/cnectweb
# DAE-3689 - Implements the General fields of the Description content type.

To test, do:

- Get the code
- Update the website configurations
- Check if the Description content type has the fields presented like the screenshot attached on the Jira ticket.

If everything's ok, merge.
